packages/binary-install/index.js

```diff
@@ -62,7 +62,13 @@ class Binary {
 
     return axios({ ...fetchOptions, url: this.url, responseType: "stream" })
       .then(res => {
-        res.data.pipe(tar.x({ strip: 1, C: this.installDirectory }));
+        return new Promise((resolve, reject) => {
+          const sink = res.data.pipe(
```

Comment:
Looks good to me! I had to double-check that pipe returns the `Writable`, but [the docs](https://nodejs.org/api/stream.html#readablepipedestination-options) indicate that it does :+1: